README.md

```diff
@@ -113,7 +113,8 @@ public struct ExampleTweaks: TweakLibraryType {
 	public static let colorTint = Tweak("General", "Colors", "Tint", UIColor.blue)
 	public static let marginHorizontal = Tweak<CGFloat>("General", "Layout", "H. Margins", defaultValue: 15, min: 0)
 	public static let marginVertical = Tweak<CGFloat>("General", "Layout", "V. Margins", defaultValue: 10, min: 0)
-	public static let featureFlagMainScreenHelperText = Tweak("Feature Flags", "Main Screen", "Show Body Text", true)
+	public static let font = Tweak<StringOption>.stringsList("General", "Layout", "Font", options: ["AvenirNext", "Helvetica", "SanFrancisco"])
```

Comment:
```suggestion
	public static let font = Tweak("General", "Layout", "Font", options: ["AvenirNext", "Helvetica", "SanFrancisco"])
```
After PR https://github.com/Khan/SwiftTweaks/pull/120 merged, the old initializer for `StringOption` was deprecated. You can now initialize a Tweak for `StringOption` like other types.

Reply:
Great!

Reply:
Ah I resolved the wrong one.